### PR TITLE
Small thing

### PR DESCRIPTION
**Pull request by BoAnd**

### CODE REVIEW

**Comment by hrjakobsen** [comment]
Hacktoberfest is good for stregsystemet 😂

File: stregsystem/views.py [hunk 1]
```diff
@@ -40,11 +40,11 @@ def __get_news():
 
 
 def __get_productlist(room_id):
-    l = (
-        make_active_productlist_query(Product.objects)
-                       .filter(make_room_specific_query(room_id))
+    lst = (
```
Comment (VirtualSatai) [inline stregsystem/views.py]: Hvorfor ikke bare return den med det samme så?

File: stregsystem/views.py [hunk 2]
```diff
@@ -40,11 +40,10 @@ def __get_news():
 
 
 def __get_productlist(room_id):
-    l = (
-        make_active_productlist_query(Product.objects)
-                       .filter(make_room_specific_query(room_id))
+    return (
```
Comment (hrjakobsen) [inline stregsystem/views.py]: The outer parentheses seems unnecessary, but it's a very minor detail

Comment (VirtualSatai) [reply stregsystem/views.py]: It's only there because Jesper has a phobia of long lines.

**Pull request merged**